pkg/blobstore/configuration/icas_blob_access_creator.go

```diff
@@ -1,12 +1,15 @@
 package configuration
 
 import (
+	"context"
+
 	"github.com/buildbarn/bb-storage/pkg/blobstore"
 	"github.com/buildbarn/bb-storage/pkg/blobstore/grpcclients"
 	"github.com/buildbarn/bb-storage/pkg/capabilities"
 	"github.com/buildbarn/bb-storage/pkg/digest"
 	"github.com/buildbarn/bb-storage/pkg/grpc"
 	pb "github.com/buildbarn/bb-storage/pkg/proto/configuration/blobstore"
+	"golang.org/x/sync/errgroup"
```

Comment:
Nit: empty line above this one, so that golang.org imports are distinct from github.com.

Rule of thumb inside the Buildbarn codebase is that all imports are grouped by second level domain, sorted in alphabetical order. I'm not sure how good of a rule it is, but it's simply the way the code is structured right now.